src/components/LHNOptionsList/LHNOptionsList.js

```diff
@@ -62,13 +132,38 @@ function LHNOptionsList({style, contentContainerStyles, data, onSelectRow, optio
      *
      * @return {Component}
      */
-    const renderItem = ({item}) => (
-        <OptionRowLHNDataWithFocus
-            reportID={item}
-            viewMode={optionMode}
-            shouldDisableFocusOptions={shouldDisableFocusOptions}
-            onSelectRow={onSelectRow}
-        />
+    const renderItem = useCallback(
+        ({item: reportID}) => {
+            const itemFullReport = reports[`${ONYXKEYS.COLLECTION.REPORT}${reportID}`] || {};
+            const itemReportActions = reportActions[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`];
+            const itemParentReportActions = reportActions[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${itemFullReport.parentReportID}`];
```

Comment:
As we know already, there was [crash](https://github.com/Expensify/App/issues/30998) after this refactor. `itemParentReportActions` was expected to be object but there's case of null.